whd/pipelines.py

```diff
@@ -17,22 +17,24 @@
 
 import concourse.enumerator
 import concourse.replicator
+import model
+import model.concourse
+import model.webhook_dispatcher
+
 
 logger = logging.getLogger(__name__)
 
 
-def update_repository_pipelines(
-    repo_url,
-    cfg_set,
-    whd_cfg,
+def replicate_repository_pipelines(
+    repo_url: str,
+    cfg_set: model.ConfigurationSet,
+    whd_cfg: model.webhook_dispatcher.WebhookDispatcherConfig,
+    validate_only: bool=False,
```

Comment:
rather pass-in (optionally) the deployed to use. or define a second function. I do not actually think adding such a switch will be beneficial, or save a lot of redundancies (but: will make code more complex, and more difficult to understand/maintain)